testGameSearch/GameService.swift

```diff
@@ -39,8 +41,10 @@ public class GameService {
                 if let d = data {
                     let gameLists = try JSONDecoder().decode([Game].self, from: d)
                       DispatchQueue.main.async {
-                        let game = gameLists
-                        //getting a yellow error here that says initialization of immutable value 'game' was never used.
+                        let game = gameLists.first
+                        
+                        completion(game)
```

Comment:
you would want to call the completion handler like a function when it's ready.

Reply:
ahhh this is what I needed I think. Why it wasn't coming through. I had tried completion(gameLists) before too 🤦‍♂️